### ISSUE TITLE
Floating panes do not disappear automatically if empty

### ISSUE THREAD

**Issue by marceltaeumel** [hpi-swa/vivide#102]
...because they never get a keyboard focus then and hence cannot lose it, which usually triggers the deletion.
